earthfile2llb/interpreter.go

```diff
@@ -134,6 +134,11 @@ func (i *Interpreter) handleBlockParallel(ctx context.Context, b spec.Block, sta
 			case "DOCKER":
 				// TODO
 			}
+		} else if stmt.If != nil || stmt.For != nil || stmt.Wait != nil {
```

Comment:
I wonder if we should also have an `else -> error` case, so that we catch any future statement types that should be handled here.